crates/wallet/src/db/mod.rs

```diff
@@ -3,23 +3,27 @@ use rusqlite::{Connection, OptionalExtension, Result, params};
 
 use crate::types::NodeUrl;
 
+pub mod balance;
 pub mod node;
 pub mod proof;
 
+pub use balance::get_all_nodes_with_balances;
+pub use balance::get_balance_for_node;
```

Comment:
do not reexport, instead use the full path when you call it `balance::get_for_all_nodes` && `balance::get_for_node`